pywinauto/controls/uiawrapper.py

```diff
@@ -40,6 +40,7 @@
 import comtypes
 
 from .. import backend
+from ..findwindows import WindowNotFoundError
```

Comment:
Maybe it's worth using `from .. import WindowNotFoundError` to align with hwndwrapper.py. Not critical. Just to prepare for removing it from `findwindows`.